test/Microsoft.AspNet.Mvc.ModelBinding.Test/Binders/FormFileModelBinderTest.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.PipelineCore;
+using Microsoft.AspNet.PipelineCore.Collections;
+using Moq;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Test
+{
+    public class FormFileModelBinderTest
+    {
+        [Fact]
+        public async Task FormFileModelBinder_ExpectMultipleFiles_BindSuccessful()
+        {
+            // Arrange
+            var bindingContext = GetBindingContext(typeof(IEnumerable<IFormFile>));
+            var binder = new FormFileModelBinder();
+
+            // Act
+            var retVal = await binder.BindModelAsync(bindingContext);
```

Comment:
don't use names like `retVal` we don't abbreviate in code. call it `result` or `success`